etc/shadowing.yaml.dist

```diff
@@ -0,0 +1,37 @@
+# This file is an example file for using DOMjudge as a shadow system.
```

Comment:
I'd propose: `This is an example configuration file for using DOMjudge as a shadow system.`